torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7827,7 +7827,7 @@ def sample_inputs_max_unpool_grad(op_info, device, dtype, requires_grad, **kwarg
 # Includes some values such that N * N won't be a multiple of 4,
 # which should ensure we test the vectorized and non-vectorized
 # kernel code paths.
-foreach_num_tensors = [20, 23] if not TEST_WITH_SLOW else [23, 30, 300]
+foreach_num_tensors = [20, 23] if not TEST_WITH_SLOW else [23, 37]
```

Comment:
Any reason for 37? I think we should test at least one tensor of size > 65536 (N > 256) to test the case of multiple multi tensor applier chunks.

Reply:
oops, thanks